glossary.md

```diff
@@ -34,5 +34,4 @@ On Linux, the namespaces from which new [container namespaces](#container-namesp
 
 [JSON]: https://tools.ietf.org/html/rfc7159
 [UTF-8]: http://www.unicode.org/versions/Unicode8.0.0/ch03.pdf
-
```

Comment:
From the [style docs][1]:

> Referenced links should be kept in two alphabetically sorted sets, a general reference section followed by a man page section.

I'm fine dropping that and going to a single alphabetically sorted list, but if we want to do that we should also update the style docs.  I'm also fine keeping the separation and the blank line here.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/style.md#links

Reply:
If styles requries, by now I have no intention to change it. Let's keep where it was.